common/txmgr/confirmer.go

```diff
@@ -939,8 +963,10 @@ func (ec *Confirmer[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Ens
 	var wg sync.WaitGroup
 	errors := []error{}
 	var errMu sync.Mutex
-	wg.Add(len(ec.enabledAddresses))
-	for _, address := range ec.enabledAddresses {
+	handleAddresses := ec.enabledAddresses
+	handleAddresses = append(handleAddresses, ec.tracker.GetAbandonedAddresses()...)
```

Comment:
Same as above comment.
No need to add them here, since this piece of code tries to do gas bump etc.
Our abandoned Txes will get rebroadcast already by the resender, so we are good.